pyproject.toml

```diff
@@ -93,7 +93,10 @@ dev = [
 evals = []
 experimental = []
 llama-index = [
-  "llama-index==0.10.44",  # always pin to a version that keeps our notebooks working
+  "llama-index==0.10.51",  # always pin to a version that keeps our notebooks working
+  "llama-index-readers-file",
+  "llama-index-llms-openai",
+  "llama-index-embeddings-openai",
```

Comment:
Need to pin these because gpt-40 is more recent.
<img width="378" alt="Screenshot 2024-07-01 at 11 11 17 AM" src="https://github.com/Arize-ai/phoenix/assets/5640648/f57ab5a2-d3bd-4256-8b8e-359bcb448eff">

Reply:
ah ok. i should pin these as well